src/util/util.js

```diff
@@ -16,3 +16,8 @@ export const noop = () => {};
 export const noargs = (f) => () => f();
 
 export const sumUnderThreshold = (list, threshold) => list.reduce((acc, i) => acc + Math.min(i, threshold), 0);
+
+export const getCsrf = () => decodeURIComponent(document.cookie.split(/;[ \t]*/)
+  .map(cookie => cookie.trim())
+  .find(cookie => cookie.startsWith('__csrf'))
+  ?.split('=')[1] || '');
```

Comment:
Since #1186 does similar cookie parsing, what do you think about adding a little utility function for that? I feel like there's enough going on in the cookie parsing that it'd be nice to have a little reusable function.